### DISCUSSION TITLE
New features in the third edition

### DISCUSSION THREAD

**Discussion by ghost** [phil-opp/blog_os#973] [Questions]
Apologies for my curiosity, but looking through the third edition folders I saw some folders with interesting names and I'm curious what is planned for the third edition besides pixel-based framebuffer. That's very exciting to me. I look forward to the your answer

**Comment by phil-opp** [comment]
Quoting my reply from [gitter](https://gitter.im/phil-opp/blog_os?at=6087d2d32c5b9d392f35ee1e) in case other people are also interested in this: 

> We will use the APIC for interrupt handling instead of the legacy PIC. I also plan to write a few more posts about creating a simple shell and some small games (probably something like snake) to make the start more interactive. And some post about debugging with GDB and QEMU is planned too.
